src/common/Header/styled.js

```diff
@@ -0,0 +1,147 @@
+import styled from 'styled-components';
+import { ReactComponent as VideoIcon } from "./VideoIcon.svg";
+import { ReactComponent as SearchIcon } from "./SearchIcon.svg";
+
+export const Wrapper = styled.header`
+    background: ${({ theme }) => theme.color.woodsmoke};
+    height: 94px;
+    display: grid;
+    gap: 80px;
+    grid-template-columns: 1.5fr 1fr 2fr;
+    padding: 35px 16px 35px 16px;
+    align-content: center;
+    color: ${({ theme }) => theme.color.white};
+    width: 100%;
+
+    @media(max-width: ${({ theme }) => theme.breakpoint.mobileMax}px){
+        height: 142px;
+        grid-template-columns: 1fr 1fr;
+        padding: 16px;
+        gap: 19px;
+    }
+`;
+
```

Comment:
Let's create GridWrapper component here so our Header can take full width space.

![image](https://user-images.githubusercontent.com/117176251/220737531-5a2d01d3-4c2c-4770-980e-c1ceee6bdbae.png)

export const GridWrapper = styled.div`
  max-width: 1368px;
  margin: auto;
  display: grid;
  gap: 16px;
  grid-template-columns: auto minmax(0, 432px);

 It will make searchbar part of header responsive

  @media (max-width: ${({ theme }) => theme.breakpoint.mobileMax}px) {
    grid-template-columns: 1fr;
    gap: 24px;
  }
`;